packages/filesystem/src/browser/file-dialog/file-dialog-service.ts

```diff
@@ -71,10 +71,10 @@ export class DefaultFileDialogService {
         return undefined;
     }
 
-    protected async getRootNode(folderToOpen?: FileStat): Promise<DirNode | undefined> {
```

Comment:
let's keep `getRootNode` to avoid breaking changes

Reply:
Done.